requirements-dev.txt

```diff
@@ -20,7 +20,8 @@ pillow
 psutil
 pydantic
 pytest
-pytest-cases
+# pytest-cases
+git+https://github.com/jayqi/python-pytest-cases@packaging-version
```

Comment:
TODO: Add an issue to change this once it is merged.

Reply:
😂 😢